pom.xml

```diff
@@ -79,7 +84,8 @@
 		<dependency>
 			<groupId>net.sf.jasperreports</groupId>
 			<artifactId>jasperreports</artifactId>
-			<version>6.1.1</version>
+			<version>4.6.0</version>
```

Comment:
Could you explain why you need Jasper 6 as dependency? While plugin is being run on target project the project provides proper Jasper version to the plugin through classpath elements. The plugin needs to know only couple classes from Jasper while it is being compiled so imho you do not need Jasper itself as "permanent" dependency.